napari/utils/colormaps/colormap_utils.py

```diff
@@ -360,7 +419,7 @@ def vispy_or_mpl_colormap(name):
 
 def increment_unnamed_colormap(
     existing: List[str], name: str = '[unnamed colormap]'
```

Comment:
trans for "unnamed colormap" ?

Reply:
👍🏼

Reply:
No since the name is the original version.